runner/runners/invoke.go

```diff
@@ -100,6 +104,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	var co snapshot.Checkout
 	checkoutCh := make(chan error)
 
+	// set up pre/postprocessors
+	for _, pp := range inv.preprocessors {
```

Comment:
Suggest we at least add an error return to pre/post, and log the func and error value if non-nil. Consider also logging the func to begin with at debug or info (e.g. "running pre func <x>")

Reply:
had that in the internal diff but i like having it here better, good call.